api/v1beta1/common.go

```diff
@@ -91,6 +92,11 @@ type DeployStrategySection struct {
 	// +kubebuilder:validation:Optional
 	// AnsibleSkipTags for ansible execution
 	AnsibleSkipTags string `json:"ansibleSkipTags,omitempty"`
+
+	// DeployIdentifier is a randomly created string created for new deploy
+	// it can be used to override a current deploy on ansibleee objects
```

Comment:
I would phrase this as:

```
// DeployIdentifier is a generated UUID set as input on OpenStackAnsibleEE resources
// so that the OpenStackAnsibleEE controller can determine job input uniqueness.
// It is generated on each new deploy request (when DeployStrategy.Deploy is changed to true).
```

or something along those lines :smile: Feel free to wordsmith.